tests/integration/kubernetes/k8s-cpu-ns.bats

```diff
@@ -12,6 +12,7 @@ setup() {
 	[ "${KATA_HYPERVISOR}" == "firecracker" ] && skip "test not working see: ${fc_limitations}"
 	[ "${KATA_HYPERVISOR}" == "dragonball" ] && skip "test not working see: ${dragonball_limitations}"
 	[ "${KATA_HYPERVISOR}" == "qemu-tdx" ] && skip "TEEs do not support memory / CPU hotplug"
+	[ "${KATA_HYPERVISOR}" == "qemu-snp" ] && skip "Memory / CPU hotplug support not implemented for SEV-SNP"
```

Comment:
Could you extend the conditional and / or use the same message for both TDX / SNP / SEV?